datamade/committee-oversight
# Static page styling

## Overview

Closes #80. This PR fleshes out a static page template style for use as an About page, Contact, Methodology, or whatever else Lugar would like to create. I was partway through this PR when Lugar's data presentation became more in the air, and I wanted to get this into PR form and finish these coding-thoughts.  It should look like this:

![screencapture-localhost-8000-about-2019-10-09-15_36_21](https://user-images.githubusercontent.com/1094243/66518380-b72c8980-eaaa-11e9-8fca-2f4a5477a897.png)


## Testing Instructions

* Delete your volume (may not be necessary): `docker-compose down --volumes`
* Start the site back up: `docker-compose up`
* Open a new window and load `hearings.dump`:  
`docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump`
* Run migrations: `docker-compose exec app python manage.py migrate`
* Load CMS content: `docker-compose run --rm app python manage.py load_cms_content`
* Load sample committee ratings: `docker-compose run --rm app python manage.py load_committeeratings`
* View `http://localhost:8000/about` and make sure it looks like the screenshot above
* Log into the site at `http://localhost:8000/cms` and create your own static page
* Make sure you can add it and delete it from the top navbar
